.github/workflows/unittest.yaml

```diff
@@ -2,14 +2,14 @@ name: Run unit test
 on: push
 
 jobs:
-  image:
-    name: Run UT for titond-backend
-    runs-on: ubuntu-latest
+    image:
```

Comment:
Can you set the tab size to 2 in your IDE? 🙇🏼‍♂️

Reply:
i fixed it! thanks